OpenRA.Game/Network/UnitOrders.cs

```diff
@@ -67,6 +67,35 @@ public static void ProcessOrder(OrderManager orderManager, World world, int clie
 						break;
 					}
 
+				case "ReplacePlayerWithBot":
+					{
+						var client = orderManager.LobbyInfo.ClientWithIndex(clientId);
+						if (client != null && world != null)
+						{
+							var player = world.FindPlayerByClient(client);
+							if (player == null || player.IsBot)
+								break;
+
+							client.State = Session.ClientState.Ready;
+
+							var admin = orderManager.LobbyInfo.Clients.FirstOrDefault(c => c.IsAdmin);
```

Comment:
Also, if the admin drops then there may also be bots that are now left without control.  Control for all of these should be switched to another player.